src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -1787,6 +1797,8 @@ class EditorOpeningEvent implements IEditorOpeningEvent {
 export interface EditorReplacement {
 	readonly editor: EditorInput;
 	readonly replacement: EditorInput;
+	// Skips asking the user for confirmation and doesn't save the document. Only use this if you really need to!
+	readonly forceReplaceDirty?: boolean;
```

Comment:
Can you do a JSDoc comment here instead so that people get this into the hover? JSDoc is `/** */`